package.json

```diff
@@ -2,11 +2,16 @@
   "name": "platform-chaos-api",
```

Comment:
i'll assume it's implicit that the comments on the other, similar ts pr you've been working on will get applied here